test/parallel/test-runner-tap-parser-stream.js

```diff
@@ -321,7 +321,7 @@ not ok 3 - test # TODO reason`,
           status: { fail: false, pass: true, todo: false, skip: true },
           id: '2',
           description: 'test',
-          reason: '',
+          reason: true,
```

Comment:
The changes in this file, and the first two changes in `test/parallel/test-runner-tap-parser.js` don't look correct. I would think the boolean would be `status.skip`, which it looks like it is on line 321. The reason should be a string.